README.md

```diff
@@ -52,6 +52,23 @@ print("Number of tokens:", len(id2vec))
 *  [3467e9ca-ec11-444a-ba27-9fa55f5ee6c1](/id2vec/3467e9ca-ec11-444a-ba27-9fa55f5ee6c1.md)
 * <default> [92609e70-f79c-46b5-8419-55726e873cfc](/id2vec/92609e70-f79c-46b5-8419-55726e873cfc.md)
 
+## id_splitter_nn
+
+Model that contains source code identifier splitter BiLSTM weights.
```

Comment:
Should be (change the code in ml accordingly):

```
Weights of the BiLSTM network to split source code identifiers.
```

Reply:
Done in ML too